<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceAddWindow</class>
 <widget class="QDialog" name="ServiceAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>214</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>214</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add service</string>
  </property>
  <widget class="QGroupBox" name="service_gBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Service</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>28</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="service_name_hLayout">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>Service name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="service_lEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="price_hLayout">
       <item>
        <widget class="QLabel" name="price_label">
         <property name="text">
          <string>Price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="price_dSpinBox">
         <property name="maximum">
          <double>999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="name_equipment_hLayout">
       <item>
        <widget class="QLabel" name="name_of_laptop_label">
         <property name="text">
          <string>Name of laptop:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="name_equipment_comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>161</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="add_service_btn">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>11</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
